zppy/e3sm_diags.py

```diff
@@ -294,6 +295,16 @@ def add_ts_dependencies(
             end_yr,
             c["ts_num_years"],
         )
+        # e3sm_to_cmip task
+        add_dependencies(
+            dependencies,
+            script_dir,
+            "e3sm_to_cmip",
+            ts_sub,  # Relies on having the same subsection name as the corresponding ts subsection!
```

Comment:
@chengzhuzhang All the dependency handling is predicated on the `e3sm_to_cmip` subtasks sharing names with the corresponding `ts` subtasks. Is that an appropriate assumption?